### DISCUSSION TITLE
Allow lightbox to be opened from link in another lightbox

### DISCUSSION THREAD

**Discussion by ninetyninew** [biati-digital/glightbox#449] [Ideas]
**Please describe the feature you want to be implemented.**
If you have some HTML content in a lightbox it would be useful to be able to open another lightbox (closing the current one open) via the usual lightbox <a> markup.

**Explain why the feature is useful**
We have lightboxes that have content where a user may wish to click through to detail in a different lightbox.

**Have you seen it somewhere else?**
Don't recall.

**Additional context**
When trying this the lightbox link does not work, it amends the URL to add the anchor, but does not close the current lightbox and open the other.

**Comment by biati-digital** [comment]
This will probably be added in V4
